happy_birthder_script_tests.py

```diff
@@ -87,6 +92,27 @@ def _wait_reminder(self):
     # Public methods
     #
 
+    def tear_down(self):
+        """Makes clean up of the testing enviroment. """
```

Comment:
* There is a typo in `enviroment`.
* Let's rewrite the docstring: `Cleans up the testing environment`